lib/dealership.rb

```diff
@@ -0,0 +1,32 @@
+class Dealership
+  attr_reader :inventory,
+              :inventory_count
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car(new_car)
+    @inventory << new_car
+    @inventory_count += 1
+  end
+
+  def has_inventory?
+    @inventory_count > 1
+  end
+
+  def cars_by_make(make)
+    selected_cars = @inventory.select do |car|
+      car.type.split(' ')[0] == make
+    end
+  end
+
+  def total_value
+    inventory_value = @inventory.map do |car|
```

Comment:
Nice job using a `.map` here. 

So you are on the right track here by calling `car.total_cost` on line 29. You wouldn't put it into square brackets though. 

See how you created the `inventory_value` variable? You would just return `inventory_value` between lines 30 and 31. 

Since `.map` returns to us an array (which you are storing in the `inventory_value` variable you can then call `.sum` on that.  

```
inventory_value = @inventory.map do |car|
   car.total_cost
end
inventory_value.sum
```